### PR TITLE
feat(pipeline): icon support (#500)

### LINKED ISSUE #500: icon support
it would be great if helix pages supported icons from an svg symbol sprite file.

i would probably for the time being propose that there is a fixed `/icons.svg` file and something like `:twitter:` gets translated to

```
<svg class="icon twitter"><use xlink:href="/icons.svg#twitter"></use></svg>
```

### PR DESCRIPTION
**Pull request by rofe**
Resolves #500

### CODE REVIEW

File: src/html/iconize.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
```
Comment (tripodsan) [inline src/html/iconize.js]: please adjust copyright year.

File: src/html/iconize.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const visit = require('unist-util-visit');
+
+const regexp = /:([a-zA-Z0-9_]+):/g;
```
Comment (tripodsan) [inline src/html/iconize.js]: what about `-` ?

File: src/html/iconize.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+function iconize({ content }) {
+  if (!content.mdast) {
+    return;
+  }
+  visit(content.mdast, (node, index, parent) => {
+    if (node.type === 'text' && regexp.test(node.value)) {
+      let text = node.value;
+      const nodes = [];
+      text.match(regexp).forEach((match) => {
```
Comment (tripodsan) [inline src/html/iconize.js]: here you execute the regexp again... I would just use `regexp.exec()`

File: icon-example.json [hunk 4]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (trieloff) [inline icon-example.json]: Delete this file.

**Pull request merged**